app/vmctl/influx/v2/adapter.go

```diff
@@ -0,0 +1,110 @@
+package v2
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	influxdb2 "github.com/influxdata/influxdb-client-go/v2"
+	"github.com/influxdata/influxdb-client-go/v2/api"
+)
+
+type Config struct {
+	Addr   string
+	Token  string
+	Org    string
+	Bucket string
+
+	Filter    Filter
+	TLSConfig *tls.Config
+}
+
+// Filter contains configuration for filtering
+// the timeseries
+type Filter struct {
```

Comment:
As far as I can see, this type is redundant. You can move 
TimeStart string
TimeEnd   string
to the Config struct